jenkins/map.jinja

```diff
@@ -5,6 +5,14 @@
         'group': 'jenkins',
         'home': '/var/lib/jenkins',
         'port': '80',
-        'server_name': None
+        'server_name': None,
```

Comment:
Yeah, sorry, haven't had a chance to look at it yet; if it's all the same to you, I'd rather fetch the blessed version from Jenkins' service and cache it somewhere we choose